3.3-nginx/nginx.conf

```diff
@@ -3,27 +3,16 @@ load_module modules/ngx_http_modsecurity_module.so;
 user ${USER};
 worker_processes 1;
 
-error_log ${ERRORLOG} ${LOGLEVEL};

```

Comment:
Explanation: This value is available from [coreruleset/modsecurity-docker:/v3-nginx/conf.d/logging.conf](https://github.com/coreruleset/modsecurity-docker/blob/master/v3-nginx/conf.d/logging.conf#L8)